kubernetes/cmsweb/services/reqmgr2ms-transferor.yaml

```diff
@@ -1,25 +1,25 @@
 kind: Service
 apiVersion: v1
 metadata:
-  name: reqmgr2ms-t
+  name: ms-transferor
   namespace: dmwm
 spec:
   selector:
-    app: reqmgr2ms-t
+    app: ms-transferor
   ports:
-    - port: 8248
-      targetPort: 8248
-      name: reqmgr2ms-t
+    - port: 8247
+      targetPort: 8247
+      name: ms-transferor
     - port: 18248
```

Comment:
same comment here. Shall we update the monitoring port as well?

Reply:
@vkuznet 

Valentin, could you please comment on port number for monitoring.

Reply:
We applied common convention to assign monitoring ports as 10000+port of the service. Therefore, if trasferor port is 8247 the monitoring port would be 18247, etc.

But please keep in mind that in order for Prometheus to start using new service metrics we also need to make changes to Prometheus configuration, see https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/monitoring/prometheus.yaml

So for all services we have appropriate section there, e.g. for DAS we have
```
      - job_name: 'das-service'
        static_configs:
            - targets: ['das.das.svc.cluster.local:18217']
```
As you can guess you need to create a section for each individual service using <srv>-service as a name, and put in targets list the service entry in a form
```
<servicename>.<servicenamespace>.svc.cluster.local:<port>
```
Therefore, I suggest that you adjust PR to add appropriate section for discussed services.